typename_derive/tests/tests.rs

```diff
@@ -12,20 +12,24 @@ extern crate typename;
 
 use typename::TypeName;
 
-#[derive(TypeName)]
+#[derive(Clone, TypeName)]
 struct UnitStruct;
 
-#[derive(TypeName)]
+#[derive(Clone, TypeName)]
```

Comment:
Just to be sure that I understand, this clone here is not necessary, right?

Reply:
You're right!  I removed the unneeded `Clone` bounds to hopefully reduce confusion.